package/android/cpp/jni/include/JniSkiaDrawView.h

```diff
@@ -46,7 +46,8 @@ class JniSkiaDrawView : public jni::HybridClass<JniSkiaDrawView>,
          makeNativeMethod("updateTouchPoints",
                           JniSkiaDrawView::updateTouchPoints),
          makeNativeMethod("registerView", JniSkiaDrawView::registerView),
-         makeNativeMethod("unregisterView", JniSkiaDrawView::unregisterView)});
+         makeNativeMethod("unregisterView", JniSkiaDrawView::unregisterView),
+         makeNativeMethod("renderToBitmap", JniSkiaDrawView::renderToBitmap)});
```

Comment:
Should it be commented out?

Reply:
I kept all of these so that it will be easy to comment in / out (from the overridable method `shouldRenderFirstFrameAsBitmap`) in the SkiaBaseView which is the flag telling if a view should or should not render this first frame.